src/SmartComponents/Applications/Applications.js

```diff
@@ -0,0 +1,35 @@
+import { Route, Switch } from 'react-router-dom';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import asyncComponent from '../../Utilities/asyncComponent';
+
+class Applications extends Component {
+
+    render() {
+        const appName = this.props.match.params.id;
+
+        const TestApp = asyncComponent(() => import(/* webpackChunkName: "TestApp" */ './TestApp'));
+        return (
+            <React.Fragment>
+                <PageHeader>
+                    <PageHeaderTitle title='Applications Settings'/>
+                    <p>{ `Settings for ${ appName }` }</p>
+                </PageHeader>
+                <Switch>
+                    <Route exact path='/applications/testApp' component={ TestApp }/>
```

Comment:
This `Route` is not really needed. We'll pull the settings from server and display it here without any nested routes.

Reply:
Sound good to me!